app/models/concerns/message_subscriptions.rb

```diff
@@ -46,13 +46,16 @@ def subscribe_users_to_message
   private
 
   def first_message_subscribers(klass)
-    roles = if klass == 'Order'
-      ['Order fulfilment', 'Order administrator']
-    elsif  ['Offer', 'Item'].include?(klass)
-      ['Reviewer', 'Supervisor']
-    else
-      []
-    end
+    roles = case klass
+      when 'Order'
+        ['Order fulfilment', 'Order administrator']
+      when 'Package'
+        ["Order administrator"]
```

Comment:
should include order fulfillment also

Reply:
Once #1044 is merged, we can use 'Stock administrator` or better still convert this to find users with specific messages permissions. Same as in https://github.com/crossroads/api.goodcity/pull/1044/files?file-filters%5B%5D=.feature&file-filters%5B%5D=.rb#diff-6063ab65baa7267cd865a29013a91297R108

Reply:
```
    if ['Offer', 'Item'].include?(object_class)
      message_permissions = ['can_manage_offer_messages']
    elsif object_class == 'Order'
      message_permissions = ['can_manage_order_messages']
    elsif object_class == 'Package'
      message_permissions = ['can_manage_package_messages']
    else
      message_permissions = []
    end
    User.joins(roles: [:permissions]).where(permissions: { name: message_permissions } ).distinct.pluck(:id)
```